### ISSUE TITLE
pytroch如何对梯度追踪张量进行inplace操作 | 光与影的个人博客

### ISSUE THREAD

**Issue by fuhaifei** [fuhaifei/MyBlogComment#28]
https://fuhaifei.github.io/2021/07/09/DL%E7%BC%96%E7%A8%8B/pytorch%E5%A6%82%E4%BD%95%E5%AF%B9%E6%A2%AF%E5%BA%A6%E8%BF%BD%E8%B8%AA%E5%BC%A0%E9%87%8F%E8%BF%9B%E8%A1%8Cinplace%E6%93%8D%E4%BD%9C/ 

 如何对梯度追踪张量进行inplace操作 别忘了梯度追踪张量必须为float Only Tensors of floating point and complex dtype can require gradients  问题来源当对设置了requires_grad=True的张量进行原地操作时，pytorch会抛出运行错误： 123>> X = torch.rand((3,4),re
